README.md

```diff
@@ -283,39 +286,71 @@ Then you will create any tenants that you need and switch to the tenants for fur
+Tenant `demo` created
 $ cfy profiles use [public_ip] --profile-name [alias-demo] -u admin -p [manager_password] -t demo
 Attempting to connect...
 Initializing profile azure...
 Initialization completed successfully
 Using manager [public_ip] with port 80
 ```
 
-### Last add your secrets:
+### Add your secrets:
 
 _Secrets are shared with tenants._
 
 #### AWS
 
 ```shell
 $ cfy secrets create aws_access_key_id -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `aws_access_key_id` created
 $ cfy secrets create aws_secret_access_key -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `aws_secret_access_key` created
 ```
 
 #### Azure
 
 ```shell
 $ cfy secrets create subscription_id -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `subscription_id` created
 $ cfy secrets create tenant_id -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `tenant_id` created
 $ cfy secrets create client_id -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `client_id` created
 $ cfy secrets create client_secret -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `client_secret` created
 ```
 
 
 #### Openstack
 
 ```shell
 $ cfy secrets create keystone_username -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `keystone_username` created
 $ cfy secrets create keystone_password -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `keystone_password` created
 $ cfy secrets create keystone_tenant_name -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `keystone_tenant_name` created
 $ cfy secrets create keystone_url -s AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+Secret `keystone_url` created
 ```
+
+### Upload plugins to the manager:
+
+**Note:** You need to download the plugin packages from (our website)[http://getcloudify.org/downloads/plugin-packages.html]:
+
+**Example:**
+
+```shell
+$ curl -O http://repository.cloudifysource.org/cloudify/wagons/cloudify-diamond-plugin/1.3.5/cloudify_diamond_plugin-1.3.5-py27-none-linux_x86_64-centos-Core.wgn
+$ curl -O http://repository.cloudifysource.org/cloudify/wagons/cloudify-diamond-plugin/1.3.5/cloudify_diamond_plugin-1.3.5-py27-none-linux_x86_64-Ubuntu-trusty.wgn
+$ curl -O http://repository.cloudifysource.org/cloudify/wagons/cloudify-openstack-plugin/2.0.1/cloudify_openstack_plugin-2.0.1-py27-none-linux_x86_64-centos-Core.wgn
+$ curl -O http://repository.cloudifysource.org/cloudify/wagons/cloudify-aws-plugin/1.4.4/cloudify_aws_plugin-1.4.4-py27-none-linux_x86_64-centos-Core.wgn
+$ cfy plugins upload cloudify_diamond_plugin-1.3.5-py27-none-linux_x86_64-centos-Core.wgn
+$ cfy plugins upload cloudify_diamond_plugin-1.3.5-py27-none-linux_x86_64-Ubuntu-trusty.wgn
+$ cfy plugins upload cloudify_openstack_plugin-2.0.1-py27-none-linux_x86_64-centos-Core.wgn
+$ cfy plugins upload cloudify_aws_plugin-1.4.4-py27-none-linux_x86_64-centos-Core.wgn
+```
+
+### Using pre-4.0 blueprint with a 4.0 manager
+
+**We will support a method for configuring managers to support blueprints writing for Cloudify <4.0.**
```

Comment:
We will support a method for configuring ~managers~ *Cloudify Manager* to support blueprints writing for Cloudify <4.0.